code/game/gamemodes/vampire/vampire_powers.dm

```diff
@@ -183,10 +183,8 @@
 				to_chat(target, "<span class='notice'>[user]'s feeble gaze is ineffective.</span>")
 			else
 				to_chat(user, "<span class='warning'>Your piercing gaze knocks out [target].</span>")
-				to_chat(target, "<span class='warning'>You find yourself unable to move and barely able to speak.</span>")
-				target.Weaken(10)
-				target.Stun(10)
-				target.stuttering = 10
+				to_chat(target, "<span class='warning'>You begin to notice your consciousness fading</span>")
```

Comment:
Did you also buff hypnotise to make targets sleep rather than freeze here?

Reply:
Oh yeah completely forgot I did that